### DISCUSSION TITLE
Change charge hour command

### DISCUSSION THREAD

**Discussion by calliou1234** [flobz/psa_car_controller#169] [Q&A] [answered]
Hello,

I used the command` http://127.0.0.1:5000/charge_hour?vin=YOURVIN&hour=18&min=0` on my e-208.

The problem now is that I can not disable it.
Every time I bring the car to the charging station, the charging process does not start and the car display shows `charging start 6 o'clock`.

How can I deactivate this again?

greetings
Thomas

**Comment by CalicodesIles** [comment, marked as answer]
Hello @calliou1234
You should be able to disable it with this command, specifying 0h00 : 

`http://127.0.0.1:5000/charge_control?vin=YOURVIN&hour=0&minute=0`
